test/unit/datastore/filestore.test.js

```diff
@@ -120,15 +124,20 @@ describe('FileStore', function() {
           'content-length': '2503'
         });
 
-      const data = await store.upload(file, {
+      const promise = store.upload(file, {
         filename: 'kinvey.png',
         public: true,
         mimeType: 'image/png'
-      });
-      expect(data).toIncludeKey('_data');
+      })
+        .then((data) => {
+          expect(data).toIncludeKey('_data');
+          expect(nock.isDone()).toEqual(true);
+          return data;
+        });
+      return promise.should.be.fulfilled;
     });
 
-    it('should resume a file upload when a 308 status code is received', async function() {
+    it('should resume a file upload when a 308 status code is received', function() {
```

Comment:
@thomasconner  just out of curiosity.. if files were failing to upload to GCS before, how did this test pass?